source/connect/csot.txt

```diff
@@ -215,7 +222,7 @@ This is the default mode for all tailable cursors,
 such as the tailable cursors returned by the ``find`` method on capped
 collections or change streams.
 
-The following code example iterates over documents in the ``mflix`` collection
+The following code example iterates over documents in a movie collection
```

Comment:
S: consider changing this example to show the collection definition or else you can just call it a sample collection. also consider changing the `imdb` field to something more common like title or year